learning_journal_basic/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding utf-8 -*-
+from webtest import TestApp as _TestApp
+import pytest
+
+
+@pytest.fixture()
+def app():
+    settings = {'sqlalchemy.url': 'sqlite db path'}
+    my_app = main({}, **settings)
+    app = _Test_App(my_app)
```

Comment:
You have nothing in this file called `_Test_App`. You're looking for `_TestApp`. Not quite sure why you needed to change the name at all here though...